### ISSUE TITLE
请问单独捕获音频有何作用

### ISSUE THREAD

**Issue by XMB-7** [polywock/globalSpeed#443]
有搜过issue, 看到作者回复说, 捕获是为了给音频施加各种特效的. 
但我发现, 诸如增益平移延迟等等音频特效, 在设置了相应的快捷键后, 都是可以直接触发的, 而不必先手动进行音频捕获然后再触发. 
所以就有疑问了, 既然如此, 那单独的捕获行为的意义何在, 是有什么音频处理之外的独特作用么?

**Comment by polywock** [comment]
>  都是可以直接触发的 

Two reasons

1. It can be used to quickly turn on/off the audio effects without resetting pitch, gain, etc. 

2. The TabCapture API, which Global Speed uses to manipulate the tab's audio, requires the extension to be invoked for that tab before capturing audio. The only way to invoke an extension is to either click on it through the toolbar, or using a browser-level shortcut key. 

A browser-level shortcut is defined through the extension's page and has the power to invoke an extension. I constrained the  the Tab Capture shortcut to be a browser-level shortcut (the other shortcuts can be either local or browser-level). 

- chrome://extensions/shortcuts
- edge://extensions/shortcuts

Short version: It's because the TabCapture API has limitations.

**Comment by XMB-7** [comment]
感谢回复, 第一条确实启发到我, 关闭音频捕获能够快速关闭各种音频特效, 也算是一种很棒的应用技巧.
